healthcheck/settings/base.py

```diff
@@ -232,9 +232,10 @@
 
 TBCONNECT_BQ_KEY_PATH = env.str("TBCONNECT_BQ_KEY_PATH", "bq_credentials.json")
 TBCONNECT_BQ_DATASET = env.str("TBCONNECT_BQ_DATASET", "wassup-165700.tbconnect")
+TBCONNECT_GROUP_ARM_ACTIVE = env.str('TBCONNECT_GROUP_ARM_ACTIVE', False)
 
 SELFSWAB_BQ_KEY_PATH = env.str("SELFSWAB_BQ_KEY_PATH", "bq_credentials.json")
-SELFSWAB_BQ_DATASET = env.str("SELFSWAB_BQ_DATASET", "wassup-165700.selfswab")
+SELFSWAB_BQ_DATASET = env.bool("SELFSWAB_BQ_DATASET", "wassup-165700.selfswab")
```

Comment:
Think you changed the wrong one here?

Reply:
I did rectify this, i will double check thanks